common/shlibs

```diff
@@ -3474,8 +3474,8 @@ libfifechan_opengl.so.0.1.5 fifechan-0.1.5_1
 libpolyclipping.so.22 clipper-6.4.2_1
 libSoapySDR.so.0.7 SoapySDR-0.7.0_1
 libeditorconfig.so.0 editorconfig-0.12.2_1
-libkid3-core.so.3.8.3 kid3-3.8.3_1
-libkid3-gui.so.3.8.3 kid3-3.8.3_1
+libkid3-core.so.3.8.4 kid3-3.8.4_1
+libkid3-gui.so.3.8.4 kid3-3.8.4_1
```

Comment:
Since nothing depends on kid3 for the moment, please consider dropping this for the next release.